### ISSUE TITLE
Trojan? 

### ISSUE THREAD

**Issue by yelkarama** [mvdan/sh#469]
Just got this alert from Windows Defender about the shfmt executable. 

![image](https://user-images.githubusercontent.com/5099215/71606994-d4ba3e00-2b43-11ea-8219-03dc7868b479.png)

Hope this issue is not present in the new versions ...

**Comment by yelkarama** [comment]
VirusTotal shows this same executable as clean:

[https://www.virustotal.com/gui/url/d006bdf1207d6b45e49f727f3e8ba216e80772383d263e62416d210274813826/detection](https://www.virustotal.com/gui/url/d006bdf1207d6b45e49f727f3e8ba216e80772383d263e62416d210274813826/detection
)

The only logical explanation I can think of is that this executable was used in some attack. In this case, the developer should research what happened, how the shfmt tool was used and implement some prevention measures.

**Comment by mvdan** [comment]
Please provide more information. Where did you get the binary from? What's its sha256 checksum? Do other Go programs classify as trojans? If you build shfmt from source yourself, does it also classify as a trojan?

> In this case, the developer should research what happened, how the shfmt tool was used and implement some prevention measures.

If somehow the download links on GitHub got tampered with, that would be bad, yes. But I don't think it's a big deal. The tool is open source, and Go builds are reproducible. If you install shfmt from a package manager, they should have built from source and provided signed or checksummed archives.

In other words, the binaries I provide here are for convenience, not security. If you want to do things the proper way, just take an extra ten seconds to build from source.

**Comment by mvdan** [comment]
Friendly ping @yelkarama.

**Comment by mvdan** [comment]
No answer; closing.

**Comment by yelkarama** [comment]
Hi
Sorry for having missed your earlier comments. 
the execuatble is indeed a Linux binary, which was included in this [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format) which is not supposed to be usable on Windows, but it made it there a part of a backup of my Linux home folder, which explains the old version number. 
What's intrigguing me is why **Windows Antivirus** classified it as **Severe** while all VirusTotal scan engines didn't. I made sure I submitted the direct url from your github releases.
It's your call to decide whether to give it more attention/time. I just wanted to inform about this alert in case it would have anu unforseen consequences..
You can re-close this issue for now, as I don't have any additional input on it.
Cheers,

**Comment by yelkarama** [comment]
Oops! I said _re-close_ as I thought GH would re-open the issue, but it didn't :)

**Comment by mvdan** [comment]
Antivirus software has false positives, and I have no reason to suspect that anyone has tampered with the binaries I provide for convenience. Like I said before, I think the true answer here are reproducible source builds. That's already possible with Go 1.13.x.
